skipper.go

```diff
@@ -194,6 +194,37 @@ type Options struct {
 	// annotation, or which an empty annotation, will be loaded too.
 	KubernetesRouteGroupClass string
 
+	// IngressLabelSelectors is a map of K8S labels to their values that must be present on a resource to be loaded
```

Comment:
Please use proper Go doc. So start the doc by the name (prefix Kubernetes) in all below, too

Reply:
Copy-paste error. Fixed.